addons/web/static/src/views/fields/parsers.js

```diff
@@ -199,5 +199,6 @@ registry
     .add("float", parseFloat)
```

Comment:
I would prefer a textual explanation than a link to a video

Reply:
I'll do that.